Gemfile

```diff
@@ -5,12 +5,16 @@ require_relative 'config/boot_settings'
 
 source 'https://rubygems.org'
 
+# rubocop:disable Bundler/DuplicatedGem
```

Comment:
Instead of disabling the cop, thoughts on using
```ruby
gem 'rails', case SETTINGS[:rails]
  when '6.0'
    '~> 6.0.3.1'
  when '6.1'
    '~> 6.1.4.4'
  else
    raise "Unsupported Ruby on Rails version configured in settings.yaml: #{SETTINGS[:rails]}"\
  end
```